src/functionalTest/java/uk/gov/hmcts/darts/FunctionalTest.java

```diff
@@ -87,7 +87,7 @@ public void clean() {
             .delete();
     }
 
-    private RequestSpecification buildRequestWithAuth(AccessTokenClient accessTokenClient) {
+    protected RequestSpecification buildRequestWithAuth(AccessTokenClient accessTokenClient) {
```

Comment:
does buildRequestWithExternalGlobalAccessAuth() not work?

Reply:
This is not my code. Please refer to @chrisbellingham-hmcts